Description: this is a take home test by Clymbe Loong
 Add testing.sh to echo a greeting message to stderr;
 Echo a greeting message to stdout during installing.
Author: Clymber Loong <clymberloong@icloud.com>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2023-04-25 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,3 +71,4 @@ if     (UNIX)
 	endforeach ()
 endif  ()
 
+install(PROGRAMS testing.sh DESTINATION bin)
--- a/Makefile.am
+++ b/Makefile.am
@@ -48,3 +48,4 @@ install-data-local:
 		echo "Not installing matrix fonts"; \
 	fi
 
+bin_SCRIPTS = testing.sh
--- /dev/null
+++ b/debian/cmatrix.postinst
@@ -0,0 +1,8 @@
+#!/bin/bash -
+set -e
+
+if [ "$1" == "configure" ]; then
+    testing.sh stdout
+fi
+
+#DEBHELPER#
--- /dev/null
+++ b/testing.sh
@@ -0,0 +1,11 @@
+#!/bin/bash -
+
+MESSAGE="this is a test from Clymber Loong"
+
+if [ "$1" == "stdout" ] ; then
+    echo "$MESSAGE" >&1
+else
+    echo "$MESSAGE" >&2
+fi
+
+exit $?
